### DISCUSSION TITLE
obs bypass

### DISCUSSION THREAD

**Discussion by talk1ng0** [danielkrupinski/Osiris#2730] [Ideas]
We want to play legally and want to receive videos

**Comment by KurastikO** [comment]
already has OBS bypass... only blocks ESP though.

**Reply by YeetingWaterbottle** [reply]
Yes, esp is blocked. And also the menu/spectator list/bomb timer is bypassed. I think there is more, you can test it out yourself.

**Reply by YeetingWaterbottle** [reply]
You can only use game capture for the esp bypass. Read [this post made by master looser](https://www.unknowncheats.me/forum/counterstrike-global-offensive/409060-obs-gamecapture.html)

**make sure to read the instructions and make sure the path is correct!!**
